src/Entity/User.php

```
@@ -0,0 +1,358 @@
+        return $this->email;
+    }
+
+    #[ORM\PreUpdate]
+    public function onPreUpdate(): void
+    {
+        $this->touch();
+    }
+
+    private function touch(): void
+    {
+        $this->updatedAt = new \DateTimeImmutable();
```

PHPMD (2.15.0)

Missing class import via use statement (line '356', column '32'). (undefined)

(MissingImport)